### DISCUSSION TITLE
reuse cards in another page

### DISCUSSION THREAD

**Discussion by cracksilver** [automadcms/discussions#37] [Q&A]
Hi there
I made some "cards" with sections. It's pretty nice but It takes a lot of work to get everything right down to the last detail.
So how I can copy a finished card from one side to another? Is there a easy way or is somewhere the code which I can save in snippets?
thx

**Comment by marcantondahmen** [comment]
In case you mean a section block, just select everything and copy it to the clipboard. You can paste it then on another page. Or duplicate it with the duplicate button and drag it somewhere with the little handle on the left side.

However, a custom template might be a good idea too.

**Comment by cracksilver** [comment]
yes I mean a section block. 
If I try to drag n drop with the little pointed handle on top left straight to another side in the sitenavigation, it doesn't work.
Copy the whole block section into another site gave me this error: Abschnitt Layout Das Display ist zu klein für die Verwendung des Blockeditors. Bitte wechsle für die Bearbeitung auf ein größeres Gerät. 
and looks like this:
![grafik](https://github.com/user-attachments/assets/dacf7482-5830-494e-9e78-2d4200272e33)

Just for understanding, I would like to copy a block section including the inside block sections an texts blocks, picture blocks and everything.
